strategy-service.ts

```diff
@@ -0,0 +1,7 @@
+import { type PositionType, type Pair, type Candle } from "./types/common";
+
+export function evaluateAllStrategies(record: { [key: Pair]: Array<Candle> }): {
+  [key: Pair]: PositionType;
+} {
+  return { pair1: "long" };
```

Comment:
It's better not to leave placeholder data in production code. You can simply return an empty object for now.